### ISSUE TITLE
[Feature request] default commit message

### ISSUE THREAD

**Issue by Mark2Mark** [Formkunft/Light-Table#7]
Proposal: When only one item is staged, offer the default message shown as a placeholder string in the commit message field `"Update <filename>"` and use that when committing and nothing was typed into the message by the user.

Similar to the Github Desktop app, which does that, it would for instance read `"Update a.ss01.glyph"`. Surely one might have better messages to put, but often I use that actually for quicker work.

When multiple files are staged, nothing changes compared to the current behaviour.

**Comment by florianpircher** [comment]
Yes, I also see the commit message as potential hurdle for people first getting started with Git.

Your example would probably read more like “Update /a.ss01”. And it would be great if there was some default for multiple files, too. And more specific messages, like “Add Intermediate layer to /someglyph” or similar, but that will have to stay a dream for some time.

**Comment by Mark2Mark** [comment]
> “Update /a.ss01”

Yes, that’s what I usually do. Would indeed be better to parse a .glyph into that style. But other files (fontinfo, ...) could just be left by their file name.

For multiple files I have no idea yet other than maybe compiling them into a list, in case it’s not too long. Or sth like "Update these glyphs:" in the commit message and followed by "/a /b /a.s01" in the additional commit note. There would have to be a limit though, to keep that message short and sensible.

The specific message idea is also nice, I assume that can quickly get out of hand. But you already implemented some sort of recognition and visual representation of things like intermediate layers, as I saw.
